mirror.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -euo pipefail
+
+# Dependencies: jq, curl, bash, rsync
+# Config settings
+OUTDIR='./repo'
+TMPDIR='./repo-tmp'
+# End of config settings
+
+rm -rf "${TMPDIR}"
+mkdir -p "${OUTDIR}" "${TMPDIR}"
+
+# TODO: Update to use the upstream repository
+RELEASE_INFO="$(curl -s https://api.github.com/repos/Doridian/archzfs/releases/latest)"
+
+FILE_INFO="$(echo "${RELEASE_INFO}" | jq '.assets | map(.browser_download_url + "|" + .updated_at) | join("\n")' -r)"
+
+for info in $FILE_INFO; do
+    url="$(echo "${info}" | cut -d'|' -f1)"
```

Comment:
this could perhaps be simplified to
```sh
url="${info%%|*}"
```
or, since we don't expect more than two fields
```sh
url="${info%|*}"
```